addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -239,11 +250,15 @@ const matchCache = async (request) => {
  * @param {FetchEvent} param0
  * @returns {Promise<Response>}
  */
-const processFetchRequest = async ({ request }) => {
+const processFetchRequest = async (request, isPrefetch) => {
```

Comment:
Please, adapt the documentation when changing the function's interface.

Reply:
Yes of course, it is done